src/main/java/conglo/Conglo.java

```diff
@@ -0,0 +1,253 @@
+package conglo;
+
+import conglo.manual.*;
+import conglo.task.*;
+import conglo.exception.*;
+import conglo.storage.*;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Conglo is a command-line task management application that allows users to
+ * add tasks, mark tasks as done or not done, and list tasks. The tasks can
+ * include ToDos, Events, and Deadlines.
+ */
+public class Conglo {
+
+    // Constants
+    protected static final String LINE_SEPARATOR = "-------------------------------------";
+
+    // Class variables
+    protected static String command;
+    private static final ArrayList<Task> taskList = TaskStorage.loadTasks();
+    protected static boolean isDelete = false;
+
+    /**
+     * The main method that starts the Conglo application. It handles user input
+     * and processes commands until the user types "bye".
+     *
+     * @param args Command-line arguments (not used).
+     */
+    public static void main(String[] args) {
+        greetUser();
+        Scanner scanner = new Scanner(System.in);
+        command = scanner.nextLine();
+        while (!command.equals("bye")) {
+            try {
+                processCommand(command);
+            } catch (CongloException e) {
+                System.out.println(e.getMessage());
+            }
+            printLineSeparator();
+            command = scanner.nextLine();
+        }
+        scanner.close();
+        sayGoodbye();
+    }
+
+    /**
+     * Prints a line separator to the console.
+     */
+    public static void printLineSeparator() {
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    private static void saveTasks() {
+        TaskStorage.saveTasks(taskList);
+    }
+
+    /**
+     * Greets the user when the application starts.
+     */
+    public static void greetUser() {
+        printLineSeparator();
+        System.out.println("Hola! I'm Conglo, the friendly task manager.");
+        printLineSeparator();
+        QuickManual.printManual();
+        printLineSeparator();
+        listTasks();
+        printLineSeparator();
+    }
+
+    /**
+     * Echoes the details of the newly added task and the current task count.
+     *
+     * @param task The task to be echoed.
+     */
+    public static void echoTask(Task task) {
+        int size = taskList.size();
+        if (isDelete) {
+            System.out.println("Okie! Task is removed from list:");
+            size--;
+        } else {
+            System.out.println("All done! Task added to list:");
+        }
+        String taskSuffix = size > 1 ? " tasks" : " task";
```

Comment:
could consider making this "easy to read" by using explicit `if ... else` block.